src/Preferences.cc

```diff
@@ -224,16 +224,15 @@ void Preferences::init() {
         initComboBox(this->comboBoxRotationY, Settings::Settings::inputRotateY);
         initComboBox(this->comboBoxRotationZ, Settings::Settings::inputRotateZ);
         initComboBox(this->comboBoxZoom, Settings::Settings::inputZoom);
-        initComboBox(this->comboBoxButton0, Settings::Settings::inputButton0);
-        initComboBox(this->comboBoxButton1, Settings::Settings::inputButton1);
-        initComboBox(this->comboBoxButton2, Settings::Settings::inputButton2);
-        initComboBox(this->comboBoxButton3, Settings::Settings::inputButton3);
-        initComboBox(this->comboBoxButton4, Settings::Settings::inputButton4);
-        initComboBox(this->comboBoxButton5, Settings::Settings::inputButton5);
-        initComboBox(this->comboBoxButton6, Settings::Settings::inputButton6);
-        initComboBox(this->comboBoxButton7, Settings::Settings::inputButton7);
-        initComboBox(this->comboBoxButton8, Settings::Settings::inputButton8);
-        initComboBox(this->comboBoxButton9, Settings::Settings::inputButton9);
+
+        for (int i = 0; i < 10; i++ ){ 
+			std::string s = std::to_string(i);
+            QComboBox* box = this->centralwidget->findChild<QComboBox *>(QString::fromStdString("comboBoxButton"+s));
```

Comment:
Indentation looks a bit off here

Reply:
This is a mix of tab and space indentation.
I now changed the settings in text editor.
Fixed indentation commited.

Reply:
Note: We actually use tab indentation. Existing spaces could be cruft left from the previous committer.

(we'll eventually make the switch to spaces; work in progress)

Reply:
thank you for the heads up. Note: We are here in a feature branch.

I would like to suggest, that we leave "code cosmetics" "as is" for now in this feature branch.
I agree that matching the indentation with the master branch and things like that have to be a requirements for the pull request from the feature branch into the master branch, but currently I would prefer focusing on code quality and features.

Reply:
Fair enough. I'll leave detail to @t-paul who currently "owns" the feature branch.

Reply:
So far I've seen git handling whitespace only changes quite well. That said, I agree with @MichaelPFrey, it's probably still better to do style cleanup separately.